### PR TITLE
issue #28191 - add more connection info to window title, update about box

### PR DESCRIPTION
**Pull request by gilmoskowitz**

### CODE REVIEW

File: loader/main.cpp [hunk 1]
```diff
@@ -44,7 +44,7 @@ int main(int argc, char* argv[])
 
   QApplication app(argc, argv);
   app.addLibraryPath(".");
-#ifndef Q_WS_MACX
+#ifndef Q_OS_MACX
```
Comment (davidbeauchamp) [inline loader/main.cpp]: this is just Q_OS_MAC if I recall correctly

File: common/common.pro [hunk 2]
```diff
@@ -24,7 +24,7 @@ OBJECTS_DIR = tmp
 MOC_DIR = tmp
 UI_SOURCES_DIR = tmp
 
-HEADERS = data.h \
+HEADERS = updaterdata.h                 \
```
Comment (davidbeauchamp) [inline common/common.pro]: extra spaces intentional?

Comment (gilmoskowitz) [inline common/common.pro]: Yes, the extra spaces are intentional to eventually line up all of the backslashes.

File: loader/main.cpp (hunk 1 above)
Comment (gilmoskowitz) [inline loader/main.cpp]: Good catch. The pull request has been updated accordingly. However, [Qt _5.7_](http://doc.qt.io/qt-5/qtglobal.html) doesn't have `Q_OS_MAC`! We'll have to switch again when we upgrade from Qt 5.5.

**Pull request merged**